tests/resume.test.ts

```diff
@@ -220,3 +220,63 @@ describe('patch resume', () => {
     ).rejects.toThrowError(ForbiddenError);
   });
 });
+
+describe('delete resume', () => {
+  test('delete resume route successfully deletes user resume', async () => {
+    const conn = await DatabaseConnection.get();
+    const member = UserFactory.fake();
+    const resume = ResumeFactory.fake({ isResumeVisible: false, user: member });
+    await new PortalState()
```

Comment:
nit: it doesn't really matter what the `isResumeFalse` attribute here is set to, so let's just exclude it from the factory call here. That way, it's clearer that the value doesn't matter since it's randomly generated, vs. specifying it and having the developer need to guess whether or not its value does matter.